Portfolio/index.css

```diff
@@ -1 +1,88 @@
-/* Write your code below for PORTFOLIO */
+/* Write your code below for PORTFOLIO */
+*{
+  
+    margin:  0px;
+    padding: 10;
+}
+.cont1{
+    background-color: #90A4AE;
+    padding: 10px;
+    color: #515050;
+  
+}
+nav ul{
+    padding: 0px;
+    margin: 0px;
+}
+nav li{
+    list-style-type: none;
+}
+.button{
+    border-radius: 15px;
+    padding: 10px;
+    margin: 10px;
+    border-color: #515050;
+}
+.contact{
+    background-color: transparent;
+}
+img{
+    border-radius: 50%;
+}
+form{
+    border: 2px solid #ffffff66;
+    background-color: #bfbfba;
+    display: inline-block;
+    padding: 5px 5px;
+    margin: 5px;
+    border-radius: 15px;

```

Comment:
Consider adding a unit to values like padding, margin, etc. For example, padding: 5px 5px;